app/scripts/controllers/eventRegistrations.js

```diff
@@ -499,36 +477,6 @@ angular
         return _.find($scope.registrations, { id: id });
       };
 
-      $scope.getRegistrationLastUpdatedTimestamp = (registrant) => {
```

Comment:
We determined that we no longer need this function, as we are now just using the ```registrant.lastUpdatedTimestamp``` value instead.